### PR TITLE
Acceptance Test for Cassandra database

### PR DESCRIPTION
**Pull request by anistal**
#### Description

Cassandra Acceptance test.
#### Reviewer

@sgomezg 
#### Test

1 acceptance test added. All passed
#### Scalastyle

All test passed.

### CODE REVIEW

File: test-at/src/test/resources/policies/ISocket-OCassandra.json [hunk 1]
```diff
@@ -0,0 +1,111 @@
+{
+  "name": "ATSocketMongo",
```
Comment (sgomezg) [inline test-at/src/test/resources/policies/ISocket-OCassandra.json]: Please, rename this. It's a little nitpick, so I'm sorry.

Comment (anistal) [inline test-at/src/test/resources/policies/ISocket-OCassandra.json]: done! :dancers:

File: test-at/pom.xml [hunk 2]
```diff
@@ -11,6 +11,17 @@
 
     <artifactId>test-at</artifactId>
 
+    <repositories>
+        <repository>
+            <id>sonatype-oss-snapshot</id>
```
Comment (sgomezg) [inline test-at/pom.xml]: Later we can upload the snapshots to internal nexus and we can skip this repo. But for now I think this is perfect :)

Only one comment. Do we want daily updates?

**Comment by sgomezg** [comment]
I like only one csv to all test, good point :)

File: test-at/pom.xml (hunk 2 above)
Comment (anistal) [inline test-at/pom.xml]: I copy directly from an EmbeddedCassandra example. My opinion is: let this lines and wait to a new Release of the package.

**Comment by sgomezg** [comment]
LGTM. Waiting for last OK of @anistal

**Pull request merged**